src/plugins/intel_npu/src/al/include/npu.hpp

```diff
@@ -80,10 +71,8 @@ class IDevice : public std::enable_shared_from_this<IDevice> {
     virtual ov::device::Type getDeviceType() const;
     virtual std::map<ov::element::Type, float> getGops() const;
 
-    virtual std::shared_ptr<SyncInferRequest> createInferRequest(
-        const std::shared_ptr<const ICompiledModel>& compiledModel,
-        const std::shared_ptr<IExecutor>& executor,
-        const Config& config) = 0;
+    virtual std::shared_ptr<SyncInferRequest> createInferRequest(std::shared_ptr<const ICompiledModel> compiledModel,
```

Comment:
Just the same issue which was highlighted in [original pr](https://github.com/openvinotoolkit/openvino/pull/26307#discussion_r1736496782) 

No visible benefit as well, plus measurable expenses added. 
Here in in other locations with similar replacements.

Reply:
same comment as above